File: apps/android/app/src/main/java/dev/tohure/android_rust_test/ui/benchmark/BenchmarkUiState.kt
package dev.tohure.android_rust_test.ui.benchmark

import androidx.compose.runtime.Immutable

@Immutable
data class BenchmarkUiState(
    val iterations: String = "1000",
    val coreP50: String = "—",
    val coreP95: String = "—",
    val nativeP50: String = "—",
    val nativeP95: String = "—",
    val isRunning: Boolean = false,
    /** Por qué no arrancó. Las otras tres apps ya lo explicaban; ésta volvía muda. */
    val error: String? = null,
)

File: apps/ios/ios-rust-test/UI/Benchmark/BenchmarkUiState.swift
struct BenchmarkUiState {
    var iterations = "1000"
    var coreP50 = "—"
    var coreP95 = "—"
    var nativeP50 = "—"
    var nativeP95 = "—"
    var isRunning = false
    /// Por qué no arrancó. Las otras tres apps ya lo explicaban; ésta volvía muda.
    var error: String?
}
